### PR TITLE
Only run edg-mkgridmap tests if the test VOMS server is sharing its

### PR DESCRIPTION
**Pull request by brianhlin**
member list.

### CODE REVIEW

File: osgtest/tests/test_51_edgmkgridmap.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ def test_01_config_mkgridmap(self):
         core.config['edg.conf'] = '/usr/share/osg-test/edg-mkgridmap.conf'
 
         core.skip_ok_unless_installed('edg-mkgridmap', 'voms-admin-server')
+        self.skip_ok_unless(core.state['voms-admin.read-members'], 'Cannot read VO member list')
```
Comment (matyasselmeci) [inline osgtest/tests/test_51_edgmkgridmap.py]: Should these be skip_ok or skip_bad?

Comment (brianhlin) [inline osgtest/tests/test_51_edgmkgridmap.py]: My only thought about not changing it to `skip_bad` is that in the future, we may have tests that only rely on `voms-admin-server` and the `edg-mkgridmap` tests will show up as failures.

**Comment by matyasselmeci** [comment]
IMO we should cross that bridge when we get to it.

**Comment by brianhlin** [comment]
Makes sense to me.

**Pull request merged**